docs/high-availability.md

```diff
@@ -0,0 +1,126 @@
+# High Availability Kubernetes Clusters
+
+## Introduction
+This document describes how to build a high-available Kubernetes cluster.  This is a fairly advanced topic, generally useful to
+people who need to set up a higly availble, redundant Kubernetes cluster for production workloads.  Users who merely want to
```

Comment:
Addressed, modified using reliable storage as "extra redundancy" rather than the default.